Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -2,8 +2,15 @@ import UIKit
 
 public protocol GestureManagerDelegate: AnyObject {
 
-    /// Informs the delegate that a gesture haas begun. Could be used to cancel camera tracking.
-    func gestureBegan(for gestureType: GestureType)
+    /// Informs the delegate that a gesture has begun. Could be used to cancel camera tracking.
+    func gestureManager(_ gestureManager: GestureManager, didBegin gestureType: GestureType)
+
+    /// Informs the delegate that a gesture has ended and whether there will be additional animations after the gesture
+    /// has completed. Does not indicate whether gesture-triggered animations have completed.
+    func gestureManager(_ gestureManager: GestureManager, didEnd gestureType: GestureType, willDecelerate: Bool)
```

Comment:
I'm wondering whether we should rename this to "willAnimate" so that it's a more accurate description for all gestures. Let's discuss this together.

Reply:
Decision: we'll rename to `willAnimate`.

Please rename the 3rd method and GestureHandlerDelegate methods accordingly